openrouteservice/src/main/java/org/heigit/ors/api/requests/centrality/CentralityRequestHandler.java

```diff
@@ -47,6 +47,8 @@ public CentralityResult generateCentralityFromRequest(CentralityRequest request)
 
         coreRequest.setBoundingBox(convertBBox(request.getBoundingBox()));
```

Comment:
I think this was taken from the `MatrixRequestHandler` - compare
https://github.com/GIScience/openrouteservice/blob/5adac66e5a8a10a194406b4b18868cc426ee15e4/openrouteservice/src/main/java/org/heigit/ors/api/requests/matrix/MatrixRequestHandler.java#L41-L42

and
https://github.com/GIScience/openrouteservice/blob/5adac66e5a8a10a194406b4b18868cc426ee15e4/openrouteservice/src/main/java/org/heigit/ors/api/requests/matrix/MatrixRequestHandler.java#L53-L54

The naming in the respective functions in the `CentralityRequestHandler` is inconsistent, though:
https://github.com/GIScience/openrouteservice/blob/5adac66e5a8a10a194406b4b18868cc426ee15e4/openrouteservice/src/main/java/org/heigit/ors/api/requests/centrality/CentralityRequestHandler.java#L21-L22
vs
https://github.com/GIScience/openrouteservice/blob/5adac66e5a8a10a194406b4b18868cc426ee15e4/openrouteservice/src/main/java/org/heigit/ors/api/requests/centrality/CentralityRequestHandler.java#L33-L34

What would your suggestion be?

Reply:
I prefer the variant `centralityRequest` because for matrix 'core' also does not seem to make sense. Maybe @HendrikLeuschner knows why it is called `core`?

Reply:
My guess would be that it is the request from the API to the 'core' of the engine. I did not write these classes, though, so no idea. Agree with Sascha that `core` naming is not useful. `centralityRequest` seems better as that is also the class name. Or just go with `request`.

Reply:
`request` would clash with the api-request that is converted/used. I'll go with `centralityRequest` :)